server/v1/controllers/shows/series/remove.js

```diff
@@ -1,16 +1,26 @@
-const db = require('../../../models');
-const { master_time_id__byShowType } = require('../utils/utils__mongoose');
+const {
```

Comment:
Many file names on the back-end are still called "show"/"shows" when they should be changed to "event"/"events" eventually. I'll wait to do this until a time when it won't cause merge conflicts (maybe when you're done with shows and as we start working on the views for traffic)